.github/ISSUE_TEMPLATE/documentation.yml

```diff
@@ -0,0 +1,28 @@
+name: Documentation
+description: Issues with documentation for the library.
+labels: 'documentation-update-needed'
+body:
+  - type: markdown
+    attributes:
+      value: |
+        ## Before Submitting Your Issue
+        If you’ve identified a problem with the documentation or samples in this repository, please consider opening a Pull Request (PR) with the necessary changes.
```

Comment:
Can 3rd party devs update our documentation? This reads to me like they can push a PR to docs.microsoft.com. Maybe something like this?

> If you’ve identified a problem with the samples in this repository, please consider opening a Pull Request (PR) with the necessary changes. Please find our contributing doc [here](https://github.com/AzureAD/microsoft-authentication-library-for-objc?tab=readme-ov-file#contribute).

> If you've identified a problem with the documentation, please share details below:

Reply:
I have put it for docs in repository, Any suggestions on making it more clearer? Should i remove it altogether Any thoughts? @mipetriu  @oldalton

Reply:
Are you referring to these docs: https://azuread.github.io/microsoft-authentication-library-for-objc/?

Reply:
Yes but it seems those are also not editable by 3rd party developers as well. my bad. I have removed the contribution part and just keeping the 'microsoft.learn.com' as placeholder